src/main/java/lesson02/CharsCounter.java

```diff
@@ -0,0 +1,45 @@
+package lesson02;
+/*
+Task 3 (CharsCounter)
+Посчитать, сколько раз символ ch встречается в строке s.
+Ограничения: нет.
+int countChars(String s, char ch)
+Входные аргументы: String s, char ch
+Тип результата: int
+ */
+
+public class CharsCounter {
+
+    public static void main (String[] args) {
+        System.out.println(countChars("There are 2 chars 'r' in this text", 'r'));
+        countCharsWithOutputMessage("Some text for test", 'S');
+        countCharsWithOutputMessage(" ", ' ');
+        countCharsWithOutputMessage("101010", '2');
+    }
+
+    private static int countChars (String inputString, char charToFind){
+        int countOfCharToFind = 0;
+        for (int i = 0; i < inputString.length(); i++) {
+            if (inputString.charAt(i) == charToFind){
+                countOfCharToFind++;
+            }
+        }
+        return countOfCharToFind;
+    }
+
+    //alternative method for chars counter with beautiful output
+    private static void countCharsWithOutputMessage (String inputString, char charToFind){
+        int countOfCharToFind = 0;
```

Comment:
You can avoid code duplication by reusing `countChars` + extra code afterward. Recall DRY principle.

Reply:
that method was removed